csv_importer/ops.py

```diff
@@ -67,18 +70,71 @@ def poll_drop(cls, context):
         return context.area
 
 
-# Register the operator and menu entry
-def menu_func_import(self, context):
-    self.layout.operator(ImportCsvPolarsOperator.bl_idname, text="CSV 🐻 (.csv)")
+class CSV_OP_ReloadData(bpy.types.Operator):
+    bl_idname = "csv.reload_data"
+    bl_label = "Reload Data"
+    bl_options = {"REGISTER", "UNDO"}
+    bl_description = (
+        "Reload the imported data file with updated values into this object"
+    )
+
+    filepath: StringProperty(  # type: ignore
+        subtype="FILE_PATH", name="File Path", description="Path to the CSV file"
+    )
+
+    def execute(self, context):
+        obj: bpy.types.Object = bpy.context.active_object  # type: ignore
+        n_points = len(obj.data.vertices)
+        update_obj_from_csv(obj, self.filepath)
+        message = f"Reloaded data for {len(obj.data.vertices)} points"
+        if len(obj.data.vertices) != n_points:
+            message += f" (was {n_points})"
+        self.report({"INFO"}, message=message)
+        return {"FINISHED"}
+
 
+def hot_reload_timer():
```

Comment:
Great idea! 
What's your feeling about how efficient this is?
Watchfiles might be an alternative https://github.com/samuelcolvin/watchfiles
(but also this is not blocking)

Reply:
It should only be a performance problem if there were _a lot_ of objects in the scene to loop over. Timer could be increased. This just uses Blender's built in system for adding periodic timers to run.